### DISCUSSION TITLE
大佬考虑过添加主动缓存的功能吗？

### DISCUSSION THREAD

**Discussion by InspoOnU** [IrineSistiana/mosdns#273] [Ideas]
开启主动缓存时，在TTL到期后自动向上游服务器查询一次。这个功能在路由器上使用效果应该比较明显。
为了避免资源浪费，可以将主动缓存的最大条目设定为缓存最大值的30%。

**Comment by IrineSistiana** [comment]
难搞，暂时没想法。

而且主动和 lazy 缓存用起来没区别。反正我是没感觉到区别。

如果真想用，建议前置一个 unbound。
